### PR TITLE
Karolina Wiaderek

### PR DESCRIPTION
**Pull request by KaroW97**

### CODE REVIEW

**Review by ArsenEloglian — APPROVED** [review]

**Comment by ArsenEloglian** [comment]
Please update attendance list

**Review by barbara-k97 — DISMISSED** [review]

**Review by ArsenEloglian — CHANGES_REQUESTED** [review]

**Comment by ArsenEloglian** [comment]
Please revert unnecessary changes.

**Comment by ArsenEloglian** [comment]
@KaroW97

**Comment by ArsenEloglian** [comment]
@KaroW97 
Please revert unnecessary changes and resolve conflicts.

File: AttendanceList.txt [hunk 1]
```diff
@@ -19,6 +26,7 @@ Kacper Węgrzynowski
   21.10.2022
 }
 Konrad Stępniak
+
```
Comment (ArsenEloglian) [inline AttendanceList.txt:29]: Please revert unnecessary changes

**Review by ArsenEloglian — CHANGES_REQUESTED** [review]

**Comment by ArsenEloglian** [comment]
@KaroW97 
what is the status of this PR?

**Review by ArsenEloglian — CHANGES_REQUESTED** [review]
The commit message is not in alignment with code diffs.

**Comment by Flehmango** [comment]
@KaroW97 closed due to inactivity. If needed, please open a new PR

**Pull request closed without merging**